receiver/elasticsearchreceiver/metadata.yaml

```
@@ -1 +1,301 @@
+    - suggest
+    - merge
+    - refresh
+    - flush
+    - warmer
+  thread_pool_name:
+    description: The name of the thread pool.
+  thread_state:
+    value: state
+    description: The state of the thread.
+    enum:
+    - active
+    - idle
+  task_state:
+    value: state
+    description: The state of the task.
+    enum:
+    - rejected
+    - completed
+metrics:
+  # these metrics are from /_nodes/stats, and are node level metrics
+  elasticsearch.node.cache.memory.usage:
+    description: The size in bytes of the cache.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+    enabled: true
+  elasticsearch.node.cache.evictions:
+    description: The number of evictions from the cache.
+    unit: "{evictions}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+    enabled: true
+  elasticsearch.node.jvm.gc.collections.count:
+    description: The number of garbage collections performed by the JVM.
+    unit: "{collections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [generation]
+    enabled: true
+  elasticsearch.node.jvm.gc.collections.elapsed:
+    description: The total time spent by the JVM running the GC.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [generation]
+    enabled: true
+  elasticsearch.node.jvm.memory.heap.used:
+    description: The current heap memory in use by the JVM.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.node.jvm.memory.heap.max:
+    description: The max memory used by the JVM.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.node.jvm.memory.nonheap.used:
+    description: The current nonheap memory in use by the JVM.
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.node.jvm.threads.count:
+    description: The number of open threads in the node's JVM process.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.node.jvm.threads.peak:
+    description: The maximum number of concurrently running threads in the node's JVM process.
+    unit: "{threads}"
+    gauge:
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.node.fs.disk.available:
+    description: The amount of disk space available across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+    enabled: true
+  elasticsearch.node.fs.io.operations:
```

```suggestion
  elasticsearch.node.fs.operations:
```
I could be wrong, but I believe the intention with [io metrics](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-naming) is that is measures an "amount" of data (i.e. bytes).

---

Looking back at this metric, I'm wondering if we should actually remove it. It counts the number of IO operations on disks visible to ES; But this is also something the hostmetrics receiver is able to count. 

Do you think we should remove it?

---

If it's not meaningfully different, we should remove it.

---

Cool, I don't think it's different enough to warrant it being here. I'll remove.